OAuth2Events.php

```diff
@@ -21,4 +21,13 @@ final class OAuth2Events
      * You could alter the access token's scope here.
      */
     public const SCOPE_RESOLVE = 'trikoder.oauth2.scope_resolve';
+
+    /**
+     * The AUTHORIZATION_REQUEST_RESOLVE event occurrs right before the system
+     * complete the authorization request.
+     *
+     * You could approve or deny the authorization request, or set the uri where
```

Comment:
> "or set the uri"

You mean or add a `RedirectResponse` to the event?

Question: If I redirect the user to a form where they accept or deny the claim. How do I get back? 
Could you add some more documentation about this?

(Im currently testing this PR)
Thank you for you work!

Reply:
Yes, I mean "or set the response, (usually a redirect response), that will be returned to the user to resolve the authorization process"

Regarding your question, you can have a listener which can extract the user decision from the request. See https://github.com/trikoder/oauth2-bundle/pull/18#issuecomment-460157339